app/controllers/api_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ApiController < ApplicationController
+  skip_before_action :verify_authenticity_token
+     private
+   def authenticated?
```

Comment:
Since you're doing authentication this way, do you need devise? If not, remove the gem from the Gemfile. :)

Reply:
I removed it, temporarily. I want to refactor authentication with devise to create a passable auth token. Like using, https://github.com/gonzalo-bulnes/simple_token_authentication. I ran into an error that I couldn't figure out.